### ISSUE TITLE
Show certificate transparency information

### ISSUE THREAD

**Issue by rugk** [sibiantony/ssleuth#85]
When this extension may finally be moved to WebExtensions, it'd also be nice to show certificate transparency information, if possible.

**Comment by sibiantony** [comment]
Yes, I believe the patch being reviewed has a `certificateTransparencyStatus`. Will add.
